### PR TITLE
Custom/tfurtado/create car lift page

### PR DESCRIPTION
**Pull request by TiFurt**

### CODE REVIEW

File: src/app/car-lift/car-lift.component.html [hunk 1]
```diff
@@ -1 +1,171 @@
...
+        <div class="division">
+          <span
+            >{{ formGroup.get('strength.force').value }}
+            {{ metric.neuton }}</span
+          >
+          <hr />
+          <span
+            >{{ formGroup.get('strength.area').value }}
+            {{ metric.meterSquare }}</span
+          >
```
Comment (biel-correa) [inline src/app/car-lift/car-lift.component.html]: Format

Comment (biel-correa) [inline src/app/car-lift/car-lift.component.html]: Format

File: src/app/car-lift/car-lift.component.html [hunk 2]
```diff
@@ -1 +1,171 @@
...
+        </div>
+      </div>
+
+      <div class="equation">
+        <span>P</span>
+        <span>=</span>
+        <span
+          >{{ formGroup.get('strength.pressure').value }}
+          {{ metric.pascal }}</span
+        >
```
Comment (biel-correa) [inline src/app/car-lift/car-lift.component.html]: Format

File: src/app/car-lift/car-lift.component.html [hunk 3]
```diff
@@ -1 +1,171 @@
...
+
+        <div class="division">
+          <span
+            >{{ formGroup.get('car.weight').value }} {{ metric.neuton }}</span
+          >
+          <hr />
+          <span
+            >{{ formGroup.get('car.area').value }}
+            {{ metric.meterSquare }}</span
+          >
```
Comment (biel-correa) [inline src/app/car-lift/car-lift.component.html]: Format

File: src/app/car-lift/car-lift.component.ts [hunk 4]
```diff
@@ -1,7 +1,610 @@
...
+
+        s.translate(this.getHalfCanvasWidth(), this.getHalfCanvasHeight());
+
+        if (window.innerWidth < 800) {
+          s.scale(0.7);
+        } else if (window.innerWidth < 1100) {
+          s.scale(0.8);
+        } else if (window.innerWidth < 1400) {
+          s.scale(0.9);
+        }
```
Comment (biel-correa) [inline src/app/car-lift/car-lift.component.ts]: Extract to method

File: src/app/car-lift/car-lift.component.ts [hunk 5]
```diff
@@ -1,7 +1,610 @@
...
+      this.scaleSize;
+
+    if (this.IsPlaying) {
+      sketch.frameRate(200);
+      if (this.strengthTubeHeight <= 0) {
+        this.pause();
+      }
+    } else {
+      sketch.frameRate(60);
+    }
```
Comment (biel-correa) [inline src/app/car-lift/car-lift.component.ts]: Add return and remove else

File: src/app/car-lift/car-lift.component.ts [hunk 6]
```diff
@@ -1,7 +1,610 @@
...
+  }
+
+  private getCanvasWidthGapResponsive(): number {
+    if (window.innerWidth < 800) {
+      return 0;
+    } else if (window.innerWidth < 1000) {
+      return 100;
+    } else if (window.innerWidth < 1400) {
+      return 200;
+    } else {
```
Comment (biel-correa) [inline src/app/car-lift/car-lift.component.ts:609]: Remove else

File: src/app/car-lift/car-lift.component.html (hunk 1 above)
Comment (TiFurt) [reply src/app/car-lift/car-lift.component.html]: done

File: src/app/car-lift/car-lift.component.html (hunk 2 above)
Comment (TiFurt) [reply src/app/car-lift/car-lift.component.html]: done

**Pull request merged**